<?xml version="1.0"?>
<launch>
  <param name="use_sim_time" value="false"/>

  <node pkg="joy" type="joy_node" name="joy_node" />
  <node pkg="jetracer" type="teleop_joy.py" name="teleop_joy_node" output="screen">
	<param name="x_speed"  value="0.3" />
	<param name="y_speed"  value="0" />
	<param name="w_speed"  value="1" />
   </node>

</launch>
